doc/lsst.synpipe/Synpipe_QuickStart.rst

```diff
@@ -0,0 +1,201 @@
+#########################
+Synpipe Quick Start Guide
+#########################
+
+Synpipe is a tool in the LSST software stack that is used to insert fake objects
+into data that is to be processed with the sofware stack. Though Synpipe is not
```

Comment:
"software"

Also, we generally use the name "LSST Science Pipelines" instead of "LSST software stack."